app/controllers/api/hardwares_controller.rb

```diff
@@ -5,7 +5,9 @@ class HardwaresController < Api::BaseController
     respond_to :json
 
     def index
-      @hardwares = policy_scope(Hardware)
+      hardwares = Hardware.all
+      hardwares = hardwares.where(user_id: params[:user_id]) if params[:user_id]
```

Comment:
params[:user_id].present? to avoid issues for mobile where they can potentially send empty user_id= param